### PR TITLE
(#59) sorted synonym table contents

### LINKED ISSUE #59: Synonym table contents should be alphabetically sorted
### Issue description
All aliases contents in the synonym table should be alphabetically sorted : 

Prod: https://www.cancer.gov/publications/dictionaries/cancer-drug/def/tretinoin
ODE: http://ncigovcdode393.prod.acquia-sites.com/publications/dictionaries/cancer-drug/def/tretinoin
> **ESTIMATE** TBD

### Steps to reproduce the issue

1.  Go to def page http://ncigovcdode393.prod.acquia-sites.com/publications/dictionaries/cancer-drug/def/tretinoin
2. See that US brand name and synonyms are not sorted alphabetically

### What's the expected result?
- sorted

### What's the actual result?
-not sorted

### Additional details / screenshot

Screen Shot 2020-10-14 at 10 42 36 AM

image (2)

### Related Tickets
* Issue #9999
* Issue #9999

### PR DESCRIPTION
**Pull request by kate-mashkina**
Closes #59

### CODE REVIEW

**Comment by kate-mashkina** [comment]
https://react-app-dev.cancer.gov/drug-dictionary-app/pr-65/def/tretinoin

**Review by karlikpj — APPROVED** [review]

File: src/components/atomic/SynonymTable/synonym-table.jsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ const SynonymTable = (aliases) => {
 	for (let i = 0; i < aliasTypes.length; i++) {
 		// get list of names from type object
 		const mapVal = sortedAliases[aliasTypes[i]];
+        mapVal.sort((a, b) => a.localeCompare(b));
```
Comment (arcepaul) [inline src/components/atomic/SynonymTable/synonym-table.jsx:57]: @bryanpizzillo should we be using language as an option for this sort?

Comment (arcepaul) [reply src/components/atomic/SynonymTable/synonym-table.jsx:57]: nm, this is english only, duh.

**Review by arcepaul — APPROVED** [review]

**Review by seyilonge-nci — APPROVED** [review]

**Comment by kate-mashkina** [comment]
@mworrest @VictoriaSunNIH  could you verify that the synonym table is now sorted - https://react-app-dev.cancer.gov/drug-dictionary-app/pr-65/def/tretinoin
https://react-app-dev.cancer.gov/drug-dictionary-app/pr-65/def/bevacizumab

**Comment by VictoriaSunNIH** [comment]
Looks good to me!  Passes UX review

CC: @mworrest , @lburack , @blilianyu

**Pull request merged**